apt/private/index.bzl

```diff
@@ -0,0 +1,120 @@
+"apt-get"
+
+load(":lockfile.bzl", "lockfile")
+
+_DEB_IMPORT_TMPL = '''\
+    http_archive(
+        name = "{name}",
+        urls = {urls},
+        sha256 = "{sha256}",
+        build_file_content = """\
+load("@rules_distroless//apt/private:dpkg_statusd.bzl", "dpkg_statusd")
+load("@rules_distroless//distroless:defs.bzl", "flatten")
+
+filegroup(
+    name = "odata",
+    srcs = ["data.tar.xz"]
+)
+
+dpkg_statusd(
```

Comment:
This might not be what we want;  newer versions of dpkg doesn't seem to recognize `status.d (./var/lib/dpkg/status.d/*)` files. only recognizes `/var/lib/dpkg/status` file. We might not do this by default.

Reply:
`status.d` is a very distroless specific status page: https://github.com/bazelbuild/rules_docker/pull/239, the tl;dr appears to be that it was selected to avoid collisions.

Reply:
Then i'll leave this to the users and not generate in the default template. They can always override this template to add more stuff.